spec/features/navigation_spec.rb

```diff
@@ -172,7 +173,8 @@
   describe "as a merchant, i see all user links" do
     it "plus a link to my merchant dashboard" do
 
-      merchant = User.create!(
+      meg = Merchant.create!(name: "Meg's Bike Shop", address: '123 Bike Rd.', city: 'Denver', state: 'CO', zip: 80203)
+      merchant = meg.users.create!(
```

Comment:
old test failing because we just had a merchant user that is now being redirected to a merchant dashboard but was created with no merchant. added a merchant and we're good to go.